autogen/agentchat/groupchat.py

```diff
@@ -75,6 +76,7 @@ def custom_speaker_selection_func(
         of times until a single agent is returned or it exhausts the maximum attempts.
         Applies only to "auto" speaker selection method.
         Default is 2.
+    - select_speaker_auto_message_transforms: (optional) the message transformations to apply to the nested select speaker agent-to-agent chat messages. Takes a TransformMessages object, defaults to None and is only utilised when the speaker selection method is "auto".
```

Comment:
If I read the arg name, I assumed I needed to pass in a list of transforms and not the capability itself. However, I can't think of a good name, maybe `select_speaker_transform_messages`? (I dropped `auto` since we don't have any other LLM based speaker selector)

Reply:
Good point, I've changed it to your suggested `select_speaker_transform_messages`, that's clearer.